test/DaySpec.hs

```diff
@@ -20,7 +30,7 @@ spec = parallel $ describe "Day" $ do
       it "returns 5" $ zellers 1  1  2000 `shouldBe` 5
       it "returns 6" $ zellers 28 12 2999 `shouldBe` 6
 
-    context "Sad Path" $ do
+    context "Sad Path" $
```

Comment:
So this one really tripped me up... From what I've read do is the syntax for whenever you're dealing with IO() but when I was writing these tests some of them wouldn't work without it. But they all seemed to work with it.    why?